examples-web/contracts/call-contract/CallContract.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
Should we pin to another solidity version to avoid the shanghai push0 problem?

Reply:
this was an issue with any version >0.8.19 so 8.0 should be fine

Reply:
^0.8.0  with the `^` will use the latest patch (past 0.8.19), which would exhibit the problem on non-shanghai chains.